packages/restapi/src/lib/chat/helpers/inbox.ts

```diff
@@ -52,7 +52,12 @@ export const getInboxLists = async (
         toDID: ''
       }
     }
-    feeds.push({ ...list, msg: message, groupInformation: list.groupInformation });
+
+    const messageResponse = await Promise.all([message])
+      .catch((e) => console.log(e.message))
+
+    feeds.push({ ...list, ...messageResponse, groupInformation: list.groupInformation })
```

Comment:
This should be
`feeds.push({ ...list, msg: message, groupInformation: list.groupInformation });`